src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,30 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: 20px 10px;
+  width: ($day-size+$day-gap)*$number-of-columns;
+
+  &__day {
+    box-sizing: border-box;
+    @include day-size-and-posotion;
+    @for $i from 1 through 31 {
+      &--#{$i}::before {
+        content: "#{$i}";
+        @include day-before-element-position;
+      }
+    }
+  }
+
+  @each $day, $day-number in $week-days {
+    &--start-day-#{$day} :nth-child(1) {
```

Comment:
I think we use nth-child for more difficult cases

Reply:
Agree, `first-child` is enough in this case